src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/JsonConverters/AnimatorJsonConverter.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace RegressionGames.StateRecorder.JsonConverters
 {
-    public class AnimatorJsonConverter : Newtonsoft.Json.JsonConverter, ITypedStringBuilderWriteable<Animator>, IBehaviourStringBuilderWritable
+    public class AnimatorJsonConverter : Newtonsoft.Json.JsonConverter, ITypedStringBuilderConverter<Animator>, IBehaviourStringBuilderWritable
```

Comment:
You can ignore everything below this file... just updates where I renamed the Interface to be more descriptive as I needed another interface to do exactly what this old name said it did....